docs/how-to/migrate-from-pytest-operator.md

```diff
@@ -81,6 +81,9 @@ In your tests, use the fixture like this:
 def test_active(juju: jubilant.Juju):
     juju.deploy('mycharm')
     juju.wait(jubilant.all_active)
+
+    # Or wait for just 'mycharm' to be active (ignore other apps):
```

Comment:
```suggestion
    # Or wait for just 'mycharm' to be active (ignoring other deployed apps):
```

or "other apps in this model"

Reply:
This one I don't think is necessary, as it's always "in this model" and apps are always deployed.